src/legacy/core_plugins/interpreter/server/lib/create_handlers.js

```diff
@@ -27,6 +27,9 @@ export const createHandlers = (request, server) => {
 
   return {
     environment: 'server',
+    // A temporary measure to allow the timelion data source to negotiate secure connections to the Kibana server.
+    // See https://github.com/elastic/kibana/pull/26809 and https://github.com/elastic/kibana/issues/26812
+    __dangerouslyUnsupportedSslConfig: server.config().get('server.ssl'),
```

Comment:
As noted by the comments, this needs to go. 

We are very careful about expanding our API, especially as it relates to `handlers`. Handlers should never contain complex undocumented objects or methods. If we can get away with having nothing at all on `handlers` we should. 

The only function we currently expose on handlers is the elasticsearch client (as wrapped by `callWithRequest`) and while we ere somewhat uncomfortable with it went ahead because the client is specced, staffed, versioned, documented, and pegged to the version of Elasticsearch that Kibana requires.

Reply:
@legrego I'm fine to stamp this PR, but I'd like a follow-up issue, as a blocker to 6.7, and mentioned here.